app/views/decidim/vocdoni/admin/census/_decidim_permissions_form.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= decidim_form_for form, url: election_census_path(election), method: :post do |form| %>
+  <%= form.hidden_field :census_permissions_form, value: true %>
+  <% current_organization.available_authorizations.each_with_index do |authorization, index| %>
+    <% checkbox_id = "verification_type_#{authorization}_#{index}" %>
+    <div class="row column">
+      <%= form.check_box :verification_types,
+                         { id: checkbox_id,
+                           multiple: true,
+                           include_hidden: false,
+                           checked: election.verification_types.include?(authorization),
+                           label: false
+                         },
+                         authorization,
+                         false %>
+      <%= form.label :verification_types, t("decidim.authorization_handlers.#{authorization}.name"), for: checkbox_id %>
+    </div>
+  <% end %>
+  </div> <!-- closing tag for card-section -->
```

Comment:
Ah, now I understand what you were mentioning in Telegram about this button.

It's ugly, but having the comment here and in the file that's calling this partial is a great mitigation to the ugliness 👌🏽